LondonBritishLibrary/orient/BLorient12859.xml

```diff
@@ -0,0 +1,437 @@
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            <msItem xml:id="ms_i19.8">
+                                <title ref="LIT7103PrayOTprophets#Hezekiah"/>
+                                <incipit xml:lang="gez">ወሶበ፡ ጸርሐ፡ ሕዝቅያስ፡ ስመከ፡ እንዘ፡ ይብል፡ ወገላውዮስ፡</incipit>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            <msItem xml:id="ms_i19.9">
+                                <title ref="LIT7103PrayOTprophets#Jonah"/>
+                                <incipit xml:lang="gez">ወሶበ፡ ጸርሐ፡ ስመከ፡ <sic>ዳዊት፡</sic> ዮናስ፡ እንዘ፡ ይብል፡ ዳላዊ፡</incipit>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            <msItem xml:id="ms_i19.10">
+                                <title ref="LIT7103PrayOTprophets#Jeremiah"/>
+                                <incipit xml:lang="gez">ወሶበ፡ ጸርሐ፡ ኤርምያስ፡ ስመከ፡ እንዘ፡ ይብል፡ <gap reason="ellipsis"/> ገደላፊል፡</incipit>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                        </msItem>
+                        <msItem xml:id="ms_i20">
+                            <title ref="LIT7130HaylaTsalai"/>
+                            <note>The text is written in the second, third or fourth column under the twelfth picture interrupting the surrounding
+                                text of <ref target="#ms_i19"/>.</note>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ጸሎት፡ በእንተ፡ ኃይለ፡ ፀላዒ፡ ወፀር፡ ወበእንተ፡ ኵናት፡ ዕንቋዕ፡ ይትፌሥሑ፡ ወይትሐሠዩ፡ 
+                                ብከ፡ ኵሎሙ፡ እለ፡ የኃሡከ፡ እግዚኦ፡ ወይበሉ፡ ዘልፈ፡ ዓቢይ፡ እግዚአብሔር፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i21">
+                            <title ref="LIT7131KwinatTsar"/>
+                            <note>The text is written in the second, third or fourth column under the twelfth picture interrupting the surrounding
+                                text of <ref target="#ms_i19"/>.</note>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ንስእለከ፡ ወናስተበቁዓከ፡ በእንተ፡ ኵናት፡ ወፀር፡ ከመ፡ ታውድቆሙ፡ ለጸርየ፡ ወለፀላእትየ፡
+                                ወከመ፡ ኢይንሥአኒ፡ ሞት፡ ዘእንበለ፡ ጌዜየ፡ ተማኅፀንኩ፡ በለዋችን፡ ስምከ፡ በአርዳድ፡ ስምከ፡ ስም፡ መግረሬ፡ ፀር፡</incipit>
+                            <textLang mainLang="gez" otherLangs="am"/>
+                        </msItem>
+                        <msItem xml:id="ms_i22">
+                            <title ref="LIT5888AsmatPrayer">Prayer against enemies</title>
+                            <note>The text is written in two columns under the fourteenth picture, the text in the first column is partly illegible.</note>
+                            <incipit xml:lang="gez">ፍዙዝ፡ ረስዮሙ፡ ለጸርየ፡ እለ፡ ይፀብዑኒ፡ አረሚ፡ ወክርስቲያን፡ ከዓውያነ፡ ደም፡ በሜሎስ፡ ገጽከ፡ በኬብሮን፡ ለግግ፡ ሕትሞሙ፡
```

Comment:
As I understand Strelcyn, the beginning of this prayer is "partly illegible" and what he (and you) reproduces is the end = the explicit